src/google/TreeMap.js

```diff
@@ -0,0 +1,158 @@
+            context.columns([['Location', 'Parent', 'Market trade volume (size)', 'Market increase/decrease (color)']]);
+            _callback();
+        }
+        
+        function setData() {
+            context.data(
+            [
+              ['Global',    null,                 0,                               0],
+              ['America',   'Global',             0,                               0],
+              ['Europe',    'Global',             0,                               0],
+              ['Asia',      'Global',             0,                               0],
+              ['Australia', 'Global',             0,                               0],
+              ['Africa',    'Global',             0,                               0],
+              ['Brazil',    'America',            11,                              10],
+              ['USA',       'America',            52,                              31],
+              ['Mexico',    'America',            24,                              12],
+              ['Canada',    'America',            16,                              -23],
+              ['France',    'Europe',             42,                              -11],
+              ['Germany',   'Europe',             31,                              -2],
+              ['Sweden',    'Europe',             22,                              -13],
+              ['Italy',     'Europe',             17,                              4],
+              ['UK',        'Europe',             21,                              -5],
+              ['China',     'Asia',               36,                              4],
+              ['Japan',     'Asia',               20,                              -12],
+              ['India',     'Asia',               40,                              63],
+              ['Laos',      'Asia',               4,                               34],
+              ['Mongolia',  'Asia',               1,                               -5],
+              ['Israel',    'Asia',               12,                              24],
+              ['Iran',      'Asia',               18,                              13],
+              ['Pakistan',  'Asia',               11,                              -52],
+              ['Egypt',     'Africa',             21,                              0],
+              ['S. Africa', 'Africa',             30,                              43],
+              ['Sudan',     'Africa',             12,                              2],
+              ['Congo',     'Africa',             10,                              12],
+              ['Zaire',     'Africa',             8,                               10]
+            ]);
+        }
+        
+        setColumns( function() {
+            setData();
+        });   
+ 
+        return this;
+    };
+    
+    TreeMap.prototype.defaultlTooltip = function(row, size, value) {
+        var data =  TreeMap.prototype.mapData;
```

Comment:
@GordonSmith `this` in this context does not refer to TreeMap.  This function is called in by the "generateTooltip" option which is specific to this widget. [docs](https://developers.google.com/chart/interactive/docs/gallery/treemap#Tooltips)  Therefore `this._data_google;`  is undefined here.   Because this option automatically passes arguments (row, size, value), I have no way to pass the TreeMap context into my function to get _data_google.  As a result I came up with adding it to TreeMap.prototype.mapData, which is accessible here.  Hope that makes sense, if you you can think of a better way to get _data_google into the function please let me know.